Tests/LibSQL/TestNonSqlStatementParsing.cpp

```diff
@@ -0,0 +1,56 @@
+/*
```

Comment:
I don't think this needs to be in its own file, it can just got in TestSqlStatementParser.cpp. It's still a SQL statement even if it's our own thing :)

Would also be nice to have a test case added to TestSqlStatementExecution.cpp to validate the execution of this statement.